config/fetcher.go

```diff
@@ -0,0 +1,14 @@
+package config
+
+// Fetcher provides an interface to get typed information out of a configuration
+// "source". These sources could be the OS enviornment, a .gitconfig, or even
+// just a `map`.
+type Fetcher interface {
+	// Get returns the string value assosicated with a given key, or an
+	// empty string if none exists.
+	Get(key string) (val string)
+
+	// Bool returns the boolean value assosicated with a given key, or an
+	// empty string if none exists.
```

Comment:
Ah, whoops. This code is actually removed in #1419, but I will apply the typo fix there 😄